wnprc_billing/resources/views/updateProgramIncomeAccount.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body onload = "getValues()">
+
+    <table id="myTable" style="border: 1px solid black; font-size: 15px; width: 800px; text-align: center">
+        <tr>
+            <th style="padding:5px; font-size:20px; text-align: center; text-decoration: underline">Variable</th>
+            <th style="padding:5px; font-size:20px; text-align: center; text-decoration: underline">Current Value</th>
+            <th style="padding:5px; font-size:20px; text-align: center; text-decoration: underline">New Value</th>
+            <th></th>
+        </tr>
+        <tr>
+            <td style="padding:5px">CreditToAccount</td>
+            <td style="padding:5px" id="ctaCell1"></td>
+            <td style="padding:5px" id="ctaCell2">
+                <form>
```

Comment:
is this form element needed here?

Reply:
You're right, I didn't realize I could use the input tag without the form tag, thank you!